src/test/java/io/openliberty/tools/intellij/it/UIBotTestUtils.java

```diff
@@ -1500,29 +1509,31 @@ public static void pasteOnActiveWindow(RemoteRobot remoteRobot, boolean homeCurs
         if (homeCursor) {
             goToLineAndColumn(remoteRobot, new Keyboard(remoteRobot), 1, 1);
         }
-        // Select the content.
```

Comment:
I see a lot of instances where you remove comments. Could you perhaps put them back?

Reply:
I also agree. The code isn't that well commented to begin with but we shouldn't be removing the few comments which explain what the test code is doing.

Reply:
I’ve added back the comments to ensure clarity and provide context for the changes.